docs/conf.py

```diff
@@ -335,7 +335,9 @@
         "reference_url": {
             "astropy": None,
             "matplotlib": "https://matplotlib.org/stable/",
-            "numpy": "https://numpy.org/doc/stable/",
+            # The numpy search js isn't loadable at the moment (2022-12-07)
```

Comment:
Wait, what?

Reply:
This was causing an error, and it's because the JS is missing some quotes apparently :man_shrugging:

Reply:
But I don't see any such failure in other PRs. Is the JS only used for this PR?